nucliadb/nucliadb/ingest/consumer/auditing.py

```diff
@@ -236,49 +236,47 @@ def iterate_auditable_fields(
     async def collect_audit_fields(
         self, message: writer_pb2.BrokerMessage
     ) -> list[audit_pb2.AuditField]:
+        if message.type == writer_pb2.BrokerMessage.MessageType.DELETE:
```

Comment:
the changes in the `collect_audit_fields` method are just checking for the message type at the beginning of the method to avoid opening and closing a transaction for nothing.